concepticondata/references/references.bib

```diff
@@ -3544,3 +3544,10 @@ @article{Speed2017
   pages={2204-2218},
   year={2017}
 }
+
+@article{Gerardi2020,
```

Comment:
Do you have volume, number and page numbers for this reference as well?

Reply:
No, it is not published yet. @LinguList asked to have this included now, updating the reference later -- the rationale is that in this case we are not making a dataset out of a published work, but preparing the dataset for a work that will be published with a CLDF dataset already (I think @xrotwang has even reserved a DOI already).

Reply:
I see. Ignore my comment then!

Reply:
Well, in fact, I'd propose to already list it as 2021, to indicate the year of the source, as we know it appeared before. So we can then write as "note" field: published online before print. @LanguageStructure, how many pages does the text have, I mean, you have seen the proofs already, right? could you share them with us or tell us already the DOI?